### DISCUSSION TITLE
Where can I find the release notes?

### DISCUSSION THREAD

**Discussion by Tejeev** [mProjectsCode/obsidian-meta-bind-plugin#380] [Q&A] [answered]
I'm trying to find the summary of changes made in the version, but the release notes appear to be black.  Are they noted somewhere else and I've just not found them, yet?

**Comment by mProjectsCode** [comment, marked as answer]
https://github.com/mProjectsCode/obsidian-meta-bind-plugin/blob/master/CHANGELOG.md
